### ISSUE TITLE
Page for adding match results

### ISSUE THREAD

**Issue by pablopb3** [fifaleaguedevs/fifaleaguedevs.github.io#61]
I think, in the future, there should be a webpage where we can add the result of the matches played. The page should have the calendar with two inputNumerics and ideally, a select field with all the players from every team, so you can easily select the scorers of the match. It would be top to get these players from the fifa API.

**Comment by loisrilo** [comment]
> It would be top to get these players from the fifa API.

Wow, fifadevs 2.0 is coming! Like this yes! :open_hands:

**Comment by charly3pins** [comment]
I totally agree with you that is better insert result from a form instead of a pull request. 
We are thinking the better way to do this, and of course the best use case is develop an app to do it in order to be faster than a website.
As lois said, this would be a fifadevs 2.0 phase because we want to do the website and the app more generic than it's now, not only for our fifadev. We are discussing the data structure to store and manage more leagues and you're welcome to discuss it with us.

Keep doing this way guys, I love this type of discussions! <3
